### ISSUE TITLE
원글 test

### ISSUE THREAD

**Issue by murinono** [murinono/murinono#1]
원글

**Comment by murinono** [comment]
이건 답글인지? 댓글인지

**Comment by murinono** [comment]
댓글인거조

**Comment by leshem8** [comment]
ㅋㅋㅋㅋ

**Comment by leshem8** [comment]
혼자 댓글쓰는 베리

**Comment by leshem8** [comment]
> 댓글인거조

이게 댓글입니다

**Comment by murinono** [comment]
> > 댓글인거조
> 
> 이게 댓글입니다

quote reply

**Comment by murinono** [comment]
> > > 댓글인거조
> > 
> > 
> > 이게 댓글입니다
> 
> quote reply

quote reply 3 times

**Comment by murinono** [comment]
> 

[I am a software engineer looking to find a remote job with a company in the USA. What is some advice for me?](https://www.quora.com/I-am-a-software-engineer-looking-to-find-a-remote-job-with-a-company-in-the-USA-What-is-some-advice-for-me)
Originally Answered: I am a software engineer looking to find a remote job with a company in the USA. Any advice for me?
tl;dr: You should try [Turing.com](http://developers.turing.com/?s=ads_developers_quora&n=top&utm_source=ads_developers_quora&utm_medium=top).

I am Vijay Krishnan, Founder & CTO of Turing.com, based in Palo Alto, California, right in the heart of the Silicon Valley.

We match exceptional software engineers from around the world to top U.S. and Silicon Valley companies that are hiring for full-time remote software positions.

100+ companies including those backed by Google, Andreessen, Founders Fund, Kleiner and Bloomberg have hired Turing engineers from 70+ countries around the world.

Work Remotely with US Software companies. No Visa Needed.
Live anywhere in the world, enjoy career growth opportunities formerly only available to engineers living in the Silicon Valley — work with former Tech Leads and Engineering Managers from Google, Facebook, Microsoft, Amazon, & various top tech companies and advance rapidly towards your professional goals
Long-term & Full-time Engagements.
Turing only partners with U.S companies that are hiring for full-time long term roles remotely and also works hard on re-matching you to subsequent long term opportunities. If you register and pass Turing’s tests and interviews, you may never have to interview for a job again.
Get the salary you deserve.
Turing developers are compensated with salaries that are better than local companies in most countries.
Find Turing Jobs in 100+ Skills - Get hired in Front-end, Back-end, Fullstack, Devops, Mobile, AI/ML/Data Engineering roles for skills such as React, Node, Python, Angular, Swift, React Native, Android, Java, Rails, Golang, PHP, Vue, Devops, Machine Learning etc.
Find Jobs Across various Levels.
Turing hires developers who are mid level, senior level engineers, tech leads, engineering managers and more.

**Comment by murinono** [comment]
> > > > 댓글인거조
> > > 
> > > 
> > > 이게 댓글입니다
> > 
> > 
> > quote reply
> 
> quote reply 3 times

[I am a software engineer looking to find a remote job with a company in the USA. What is some advice for me?](https://www.quora.com/I-am-a-software-engineer-looking-to-find-a-remote-job-with-a-company-in-the-USA-What-is-some-advice-for-me)
Originally Answered: I am a software engineer looking to find a remote job with a company in the USA. Any advice for me?
tl;dr: You should try [Turing.com](http://developers.turing.com/?s=ads_developers_quora&n=top&utm_source=ads_developers_quora&utm_medium=top).

I am Vijay Krishnan, Founder & CTO of Turing.com, based in Palo Alto, California, right in the heart of the Silicon Valley.

We match exceptional software engineers from around the world to top U.S. and Silicon Valley companies that are hiring for full-time remote software positions.

100+ companies including those backed by Google, Andreessen, Founders Fund, Kleiner and Bloomberg have hired Turing engineers from 70+ countries around the world.

Work Remotely with US Software companies. No Visa Needed.
Live anywhere in the world, enjoy career growth opportunities formerly only available to engineers living in the Silicon Valley — work with former Tech Leads and Engineering Managers from Google, Facebook, Microsoft, Amazon, & various top tech companies and advance rapidly towards your professional goals
Long-term & Full-time Engagements.
Turing only partners with U.S companies that are hiring for full-time long term roles remotely and also works hard on re-matching you to subsequent long term opportunities. If you register and pass Turing’s tests and interviews, you may never have to interview for a job again.
Get the salary you deserve.
Turing developers are compensated with salaries that are better than local companies in most countries.
Find Turing Jobs in 100+ Skills - Get hired in Front-end, Back-end, Fullstack, Devops, Mobile, AI/ML/Data Engineering roles for skills such as React, Node, Python, Angular, Swift, React Native, Android, Java, Rails, Golang, PHP, Vue, Devops, Machine Learning etc.
Find Jobs Across various Levels.
Turing hires developers who are mid level, senior level engineers, tech leads, engineering managers and more.

**Comment by murinono** [comment]
> > > > > 댓글인거조
> > > > 
> > > > 
> > > > 이게 댓글입니다
> > > 
> > > 
> > > quote reply
> > 
> > 
> > quote reply 3 times
> 
> [I am a software engineer looking to find a remote job with a company in the USA. What is some advice for me?](https://www.quora.com/I-am-a-software-engineer-looking-to-find-a-remote-job-with-a-company-in-the-USA-What-is-some-advice-for-me) Originally Answered: I am a software engineer looking to find a remote job with a company in the USA. Any advice for me? tl;dr: You should try [Turing.com](http://developers.turing.com/?s=ads_developers_quora&n=top&utm_source=ads_developers_quora&utm_medium=top).
> 
> I am Vijay Krishnan, Founder & CTO of Turing.com, based in Palo Alto, California, right in the heart of the Silicon Valley.
> 
> We match exceptional software engineers from around the world to top U.S. and Silicon Valley companies that are hiring for full-time remote software positions.
> 
> 100+ companies including those backed by Google, Andreessen, Founders Fund, Kleiner and Bloomberg have hired Turing engineers from 70+ countries around the world.
> 
> Work Remotely with US Software companies. No Visa Needed. Live anywhere in the world, enjoy career growth opportunities formerly only available to engineers living in the Silicon Valley — work with former Tech Leads and Engineering Managers from Google, Facebook, Microsoft, Amazon, & various top tech companies and advance rapidly towards your professional goals Long-term & Full-time Engagements. Turing only partners with U.S companies that are hiring for full-time long term roles remotely and also works hard on re-matching you to subsequent long term opportunities. If you register and pass Turing’s tests and interviews, you may never have to interview for a job again. Get the salary you deserve. Turing developers are compensated with salaries that are better than local companies in most countries. Find Turing Jobs in 100+ Skills - Get hired in Front-end, Back-end, Fullstack, Devops, Mobile, AI/ML/Data Engineering roles for skills such as React, Node, Python, Angular, Swift, React Native, Android, Java, Rails, Golang, PHP, Vue, Devops, Machine Learning etc. Find Jobs Across various Levels. Turing hires developers who are mid level, senior level engineers, tech leads, engineering managers and more.

[I am a software engineer looking to find a remote job with a company in the USA. What is some advice for me?](https://www.quora.com/I-am-a-software-engineer-looking-to-find-a-remote-job-with-a-company-in-the-USA-What-is-some-advice-for-me)
Originally Answered: I am a software engineer looking to find a remote job with a company in the USA. Any advice for me?
tl;dr: You should try [Turing.com](http://developers.turing.com/?s=ads_developers_quora&n=top&utm_source=ads_developers_quora&utm_medium=top).

I am Vijay Krishnan, Founder & CTO of Turing.com, based in Palo Alto, California, right in the heart of the Silicon Valley.

We match exceptional software engineers from around the world to top U.S. and Silicon Valley companies that are hiring for full-time remote software positions.

100+ companies including those backed by Google, Andreessen, Founders Fund, Kleiner and Bloomberg have hired Turing engineers from 70+ countries around the world.

Work Remotely with US Software companies. No Visa Needed.
Live anywhere in the world, enjoy career growth opportunities formerly only available to engineers living in the Silicon Valley — work with former Tech Leads and Engineering Managers from Google, Facebook, Microsoft, Amazon, & various top tech companies and advance rapidly towards your professional goals
Long-term & Full-time Engagements.
Turing only partners with U.S companies that are hiring for full-time long term roles remotely and also works hard on re-matching you to subsequent long term opportunities. If you register and pass Turing’s tests and interviews, you may never have to interview for a job again.
Get the salary you deserve.
Turing developers are compensated with salaries that are better than local companies in most countries.
Find Turing Jobs in 100+ Skills - Get hired in Front-end, Back-end, Fullstack, Devops, Mobile, AI/ML/Data Engineering roles for skills such as React, Node, Python, Angular, Swift, React Native, Android, Java, Rails, Golang, PHP, Vue, Devops, Machine Learning etc.
Find Jobs Across various Levels.
Turing hires developers who are mid level, senior level engineers, tech leads, engineering managers and more.
